### PR TITLE
Update launch guide and related content - Closes #1027

### LINKED ISSUE #1027: Add guide how to launch a blockchain network
A guide which focuses on explaining how to create an initial blockchain network of several nodes, and best practices how to decentralize the network.

Parent issue: #841

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1027

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/genesis-block.adoc [hunk 1]
```diff
@@ -73,8 +73,15 @@ Creating genesis block [===========         ] 55% 44.6s
 
 After completion, the new genesis block will be copied to `config/default/`, where it automatically replaces the outdated genesis block.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/genesis-block.adoc]:
```suggestion
After completion, the new genesis block will be copied to `config/default/`, whereby it automatically replaces the outdated genesis block.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 2]
```diff
@@ -1,13 +1,15 @@
-= Launch a blockchain application
+= Launching a blockchain application
 Mona Bärenfänger <mona@lightcurve.io>
 :description: How to launch a blockchain application together with the components, seed nodes, genesis block, config objects, & also adding additional nodes.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
:description: How to launch a blockchain application together with the components, seed nodes, genesis block, config objects, and also adding additional nodes.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 3]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-The network will consist of many different nodes that store the blockchain, create new blocks, and update each other about events in the network.
-
-Nodes are servers that are connected to a blockchain network.
-These servers have a <<node_app,node application>> installed that handles all communication with other nodes in the network.
-
-After receiving a new event and performing the default validations, a node will pass on the information to a set of peer nodes in its peer list.
+How to turn your local 1-node Devnet into an actual blockchain network.
 
-=== Node roles
+This guide describes how to release a blockchain application to the public and how to establisha decentralized network of nodes and validators who together are maintaining and securing the blockchain application.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
This guide describes how to release a blockchain application to the public, and how to establish a decentralized network of nodes and validators that together, are both maintaining and securing the blockchain application.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 4]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+====
+This guide describes only the most important steps for releasing a basic blockchain application with one connected frontend for a public Testnet.
 
-Seed nodes::
-The seed node is a node that is specified in the xref:{url_reference_config}[config] of the node application under `modules.network.seedPeers`.
-Apart from this, the seed node is considered a normal node, and exhibits the same properties as all of the other nodes.
-+
-Newly connected nodes will use the seed peers in their config as a start point to discover the rest of the network.
-Hence, this is the reason why seed nodes should always be both reachable and connected to the network.
+Additional steps might be required for more complex blockchain applications (e.g. releasing additional software like Middllewares beetween frontend and blockchain application).
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
Additional steps might be required for more complex blockchain applications (e.g. releasing additional software such as Middllewares between the frontend and the blockchain application).
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 5]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-Forging nodes::
-As the name suggests, forging nodes are nodes that have xref:{url_guides_forging}[forging enabled] for a delegate in the network.
-In order to actively forge, the delegate has to be present in the top 101 delegate list.
+== Initial state: Devnet
 
-API nodes::
-API nodes are nodes that offer xref:{url_guides_api}[API access] to specific whitelisted IPs or to the general public.
-They are used by client applications to interact with the blockchain network.
+The Devnet is used as default network to connect to during the development of a blockchainn application.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
The Devnet is used as the default network to connect to during the development of a blockchain application.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 6]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+Starting the node after xref:{url_guides_setup}[Bootstrappinng the blockchain application] will always connect to a local Devnet.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
Starting the node after xref:{url_guides_setup}[Bootstrapping the blockchain application] will always connect to a local Devnet.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 7]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-=== Components of a blockchain application
+During development it is both necessary and convenient to have a rather centralized network with only one node connected.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
During development, it is both necessary and convenient to have a rather centralized network with only one node connected.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 8]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-The development and publishing of the client application is similar to a regular web application.
+To turn the Devnet into a public Testnet, the following steps should be considered:
 
-[TIP]
-====
-There can also be several client applications that work together by communicating with the same network.
-For example, there can be an additional application that connects a particular IoT device to the blockchain.
-====
+* <<create-a-new-genesis-block,Create a new unique genesis block>> for the Testnet
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
* <<create-a-new-genesis-block,Create a new unique genesis block>> for the Testnet.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 9]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+** Create new genesis delegates & keep their credentials private and secure.
+* <<update-the-configuration-file>>
+** Update community identifier here
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
** Update community identifier here.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 10]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+=== Define seed nodes
 
-[[config]]
-== Create a configuration object
+Each node that is connecting to the network for the first time, will initially connect to the seed nodes.
+Starting from a seed node, the new node will discover the rest of the network by requesting their peer list.
+This is then followed by the peer lists of the newly discovered peers and so on.
 
-Exchange the {url_github_sdk_configDevnet}[configDevnet^] object that was passed to the node during the development with the customized version.
+To define a node as a seed node, simply add the IP and port of the seed node to the config as described in section <<config,create a configuration object>>.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc:212]:
```suggestion
To define a node as a seed node, simply add the IP and port of the seed node to the config as described in section <<config, create a configuration object>>.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 11]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-----
+Include the data of the new genesis delegates in the config (see xref:{url_guides_genesisblock_enableforging}[Enabling forging for the new genesis delegates]).
 
-[NOTE]
-====
-By default, the forging delegates list in the config is empty.
-Only the exposed `configDevnet` object includes forging data for all 101 genesis delegates.
-The user who will later deploy the node on a server can then add the credentials to the config, to xref:{url_guides_forging}[enable forging] for a particular delegate on this node.
-====
+Beside this, seed nodes require no further special config options.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
Besides this, seed nodes require no further special config options.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 12]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+Especially, it is recommended to leave the public API disabled on a seed node.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
It is highly recommended to leave the public API disabled on a seed node.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 13]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+│   │   ├── modules.ts
+│   │   ├── plugins/
+│   │   └── plugins.ts
+│   └── commands/
+├── test/
+└── README.md
+----
 
-TIP: How to replace the genesis delegates with real delegates is covered in the next section <<more_nodes, Add more nodes>>.
+It is recommended to provide additional application specific documentation for users, for example by updating the README.md with all important steps to spin up a node.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
It is recommended to provide additional application specific documentation for users, for example by updating the README.md with all the important steps required to spin up a node.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 14]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-IMPORTANT: Do not forget to open the corresponding xref:{url_setup_ports}[ports] for HTTP and WS communication!
+image:guides/launch/testnet-node.png[Two nodes diagram]
 
-By default, the API access of a node is disabled.
-So after successful setup of the node, it is required to xref:{url_guides_config}[whitelist specific clients] in the config, or to make the xref:{url_guides_config}[API access public] for everyone alternatively.
+=== Add a node with enabled API
 
-Once a new node is set up, update the API endpoint in the <<client_app,client app>>, to target the new node:
+As the seed node should have RPC API disabled, add a new node to the network with API enabled.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
As the seed node should have the RPC API disabled, add a new node to the network with the API enabled.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 15]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+This node has mainly the purpose of providing blockchain data to external tools and services via the API.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
The main function of this node is to provide blockchain data to external tools and services via the API.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 16]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+Forging should bbe disabled on this node.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
Forging should be disabled on this node.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 17]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+    }
+  }
+}
+----
 
 [[more_nodes]]
-== Add more nodes
-
-During development of the blockchain application, one node was enabled for forging for all 101 genesis delegates.
+== Become validator for the Testnet
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
== Become a validator for the Testnet
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 18]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-After the release of the first version of the blockchain application, it is necessary that real delegates take the forging slots of the genesis delegates.
+After the Testnet-release of the blockchain application, it is necessary that real, independent delegate accounts take the forging slots of the genesis delegates.
 
-NOTE: The network will become decentralized for the first time when at least 51 individual delegates are actively forging in the network.
+NOTE: The network will become decentralized for the first time when at least 51% of the delegates are actively forging in the network are independent delegates.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
NOTE: The network will become decentralized for the first time when at least 51% of the delegates that are actively forging in the network, are independent delegates.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 19]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-.A forging node joins the network
-image:launch-3_nodes.png[3 nodes diagram]
+.Forging nodes join the network
+image:guides/launch/testnet-delegate-nodes.png[3 nodes diagram]
 
 To join the network as a new delegate, follow the steps listed below:
 
 . Create an own, private account on the network.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
. Create your own, private account on the network.
```

File: modules/ROOT/pages/guides/app-development/launch.adoc [hunk 20]
```diff
@@ -17,294 +19,316 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-Later, when the majority of the existing tokens are distributed amongst the different private accounts, the new delegate needs to gain the trust of the community in order to be voted into a forging position.
-====
+.. xref:{url_references_cli_createaccount}[Generate the account credentials]
+.. Use the Testnet Faucet to receive some initial tokens.
+. xref:{url_guides_forging_register}[Register a delegate].
+. Set up a node: Follow the steps in the `README` file or the documentation of the blockchain application.
+Normally it involves downloading the application code from a public source, configuring it (via config or flags on startup), and starting the application.
+. xref:{url_guides_forging}[Enable forging] for the newly registered delegate.
+. Perform a self-vote.
+. Optional: Convince other accounts to vote for your delegate, to increase the delegate vote weight if necessary.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
. Optional: Convince other accounts to vote for your delegate, in order to increase the delegate vote weight if necessary.
```

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 21]
```diff
@@ -38,6 +38,77 @@ This is achieved by choosing a unique delegate name, and posting a corresponding
 
 There are many possibilities to post the transaction, use the one you prefer: {url_lisk_wallet}[Lisk Desktop^], xref:{url_elements}[Lisk Elements], or the xref:{url_framework_dashbobardplugin}[].
 
+One way is to create the transaction object with the application CLI:
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/forging.adoc]:
```suggestion
One option is to create the transaction object with the application CLI:
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: modules/ROOT/pages/guides/app-development/launch.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/app-development/launch.adoc]:
```suggestion
This guide describes how to release a blockchain application to the public, and how to establish a network of nodes and validators that are maintaining and securing the blockchain application in a decentralized manner.
```

**Review by shuse2 — APPROVED** [review]

**Pull request merged**